src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs

```diff
@@ -61,6 +62,7 @@ public class CollectionView : RecyclerView
         /// <summary>
         /// Binding Property of selected items list in multiple selection.
         /// </summary>
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
If you want to open the property, please remove [EditorBrowsable(EditorBrowsableState.Never)].